components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -122,10 +122,11 @@ const ContributorHighlightCard = ({
   const [host, setHost] = useState("");
   const [dropdownOpen, setDropdownOpen] = useState(false);
   const { follow, unFollow, isError } = useFollowUser(
-    dropdownOpen && loggedInUser && loggedInUser?.user_metadata.user_name !== user ? user : ""
+    dropdownOpen && loggedInUser && loggedInUser.user_metadata.user_name !== user ? user : ""
```

Comment:
nit: should be OK because if the user is not logged in, `undefined` will still not equal `user`.

```suggestion
    dropdownOpen && loggedInUser?.user_metadata.user_name !== user ? user : ""
```